<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Filter</class>
 <widget class="QWidget" name="Filter">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>COMP_hack Capture Grep - Packet Filter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="listLayout">
     <item>
      <widget class="QGroupBox" name="whiteBox">
       <property name="title">
        <string>White List</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QListWidget" name="whiteList">
          <property name="toolTip">
           <string>Add commands here to always display them.</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="whiteLayout">
          <item>
           <widget class="QPushButton" name="whiteAdd">
            <property name="toolTip">
             <string>Add a command to the white list.</string>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="whiteRemove">
            <property name="toolTip">
             <string>Remove a command from the white list.</string>
            </property>
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="blackBox">
       <property name="title">
        <string>Black List</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListWidget" name="blackList">
          <property name="toolTip">
           <string>Add commands here to never display them.</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="blackLayout">
          <item>
           <widget class="QPushButton" name="blackAdd">
            <property name="toolTip">
             <string>Add a command to the black list.</string>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="blackRemove">
            <property name="toolTip">
             <string>Remove a command from the black list.</string>
            </property>
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
